modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/RankEvalSpec.java

```diff
@@ -122,10 +114,6 @@ public void writeTo(StreamOutput out) throws IOException {
             entry.getValue().writeTo(out);
         }
         out.writeVInt(maxConcurrentSearches);
-        out.writeInt(indices.size());
-        for (String index : indices) {
-            out.writeString(index);
-        }
```

Comment:
This should me handled by in.readStringArray() in the request now, and shouldn't change the wire protocol so no version-dependent logic needed here I think.